x-pack/plugins/security/public/session/session_timeout.test.tsx

```diff
@@ -103,8 +103,8 @@ describe('warning toast', () => {
     expect(http.get).not.toHaveBeenCalled();
     const toastInput = notifications.toasts.add.mock.calls[0][0];
     expect(toastInput).toHaveProperty('text');
-    const reactComponent = (toastInput as any).text;
-    const wrapper = mountWithIntl(reactComponent);
+    const mountPoint = (toastInput as any).text;
+    const wrapper = mountWithIntl(mountPoint.__reactMount__);
```

Comment:
the fact that we keep a reference to the react element in the mountpoint for serialization actually allows to do that. As the enzyme API is way better than manually mounting the mountpoint in tests (even if it does work), I think this trick is actually a good thing here.